packages/widget/src/state/swapExecutionPage.ts

```diff
@@ -165,6 +165,14 @@ export const setSwapExecutionStateAtom = atom(null, (get, set) => {
         { ...txInfo, explorerLink, status: undefined },
         transactionHistoryIndex,
       );
+
+      set(setTransactionHistoryAtom, transactionHistoryIndex, {
+        route: route as RouteResponse,
+        transactionDetails: [txInfo],
+        timestamp: Date.now(),
+        status: "unconfirmed",
+      });
+
```

Comment:
This is rendundant, because we already did it in line 163-167.

Reply:
you're right, i misread that function and thought it didn't take the txInfo and put it in transactionDetails, but it actually does